ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -64,7 +64,7 @@ Verify RHODS User Groups
 
 Verify Culler is Enabled
     [Documentation]    Verify Culler Configuration after the upgrade
-    [Tags]      Upgrade
+    [Tags]      Upgrade    Dashboard
```

Comment:
This is not a UI test

Reply:
It checks a configmap through oc, but the configmap is created and written into only by the Dashboard, so although the test is not through-ui, it is testing dashboard

https://github.com/red-hat-data-services/ods-ci/blob/78b203b5cf97de9c3175b2d64cd0e149de942153/ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot#L65-L73